.github/workflows/lint-build.yml

```diff
@@ -9,12 +9,14 @@ jobs:
         with:
           node-version: '18'
           cache: 'yarn'
-      - name: Set up MySQL
+      - name: Set up PostgreSQL
         run: |
-          sudo /etc/init.d/mysql start
-          mysql -e 'CREATE DATABASE checkpoint;' -uroot -proot
+          sudo systemctl start postgresql
```

Comment:
Was this tested? Surprised that ubuntu image on Actions has PostgreSQL installed by default 🤔